### ISSUE TITLE
Feature Request: save/copy an entire conversation

### ISSUE THREAD

**Issue by ssvenn** [AugustDev/enchanted#82]
Right now you are limited to copy one message at a time if you want to save an interesting session, and you have to manually add the User: and Assistant: headings for each message. 
It would be a nice addition if there was a Save/Copy menu option in the conversation history list context menu that currently just lets you Delete them.

**Comment by myildi** [comment]
Yes! Enchanted is really very nice to use, but the possibility of exporting the whole conversation would be very useful. Currently, we cannot even copy the question with the answer as far as I can see.
